packages/packages/api/src/medical/client/metriport.ts

```diff
@@ -356,14 +356,16 @@ export class MetriportMedicalApi {
   /**
    * Returns a URL that can be used to download the document.
    *
-   * @param fileName The file name of the document in s3.
-   * @return presigned url that expires in 20 seconds
+   * @param req.query.fileName The file name of the document in s3.
+   * @param req.query.conversionType The doc type to convert to. Valid values are "html" and "pdf".
+   * @return presigned url
    */
   //eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async getDocumentUrl(fileName: string): Promise<{ url: string }> {
+  async getDocumentUrl(fileName: string, conversionType: string): Promise<{ url: string }> {
```

Comment:
conversionType should be optional

Reply:
also might as well make it a type literal and/or enum (ie html/pdf)

Reply:
[union types](https://github.com/metriport/metriport/pull/407#discussion_r1212515214) 🙏